app/controllers/course/statistics/assessments_controller.rb

```diff
@@ -2,20 +2,22 @@
 class Course::Statistics::AssessmentsController < Course::Statistics::Controller
   include Course::UsersHelper
   include Course::Statistics::SubmissionsConcern
+  include Course::Statistics::UsersConcern
 
   def main_statistics
     @assessment = Course::Assessment.where(id: assessment_params[:id]).
                   calculated(:maximum_grade, :question_count).
-                  preload(lesson_plan_item: [:reference_times, personal_times: :course_user],
-                          course: :course_users).first
+                  preload(course: :course_users).first
     submissions = Course::Assessment::Submission.where(assessment_id: assessment_params[:id]).
                   calculated(:grade, :grader_ids).
-                  preload(:answers, creator: :course_users)
-    @course_users_hash = preload_course_users_hash(current_course)
+                  preload(creator: :course_users)
+    @course_users_hash = preload_course_users_hash(@assessment.course)
```

Comment:
Why change this?

Reply:
even though both are referring to the same thing (the assessment_params[:id] is referring to the assessment, which belongs current course), but using `@assessment.course` communicates better to the developer, since we focus on `@assessment` in this `main_statistics`. Using `current_course`, I realised, might create the confusion since it makes the previous line before this (`@assessment`) completely unrelated to this one